core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -4990,3 +4993,16 @@ private[fs2] trait StreamLowPriority {
         }
     }
 }
+
+private[fs2] trait StreamLowPriority1 {
```

Comment:
I'm moderately sure I tried this already and it'll fail the laws :(

Reply:
@kubukoz, you mean there is no lawful `Alternative` instance for `Stream`?

Reply:
See here https://github.com/functional-streams-for-scala/fs2/pull/1418

Reply:
and discussion here https://github.com/functional-streams-for-scala/fs2/issues/1394

Reply:
I see, thanks!